aws_run_celery.py

```diff
@@ -2,11 +2,21 @@
 from app import notify_celery, create_app
 from credstash import getAllSecrets
 import os
+from config import configs
+
+default_env_file = '/home/ubuntu/environment'
+environment = 'live'
+
+if os.path.isfile(default_env_file):
+    environment_file = open(default_env_file, 'r')
```

Comment:
with open(default_env_file, 'r') as environment_file:
    environment = environment_file.readline().strip()

This handles the the closing of the file as well. Python context managers :)